### ISSUE TITLE
Tensorflow vs Pytorch Activation Functions

### ISSUE THREAD

**Issue by brianjsl** [aiff22/PyNET-PyTorch#25]
Why is Sigmoid (with no coefficients) being used in the [Pytorch](https://github.com/aiff22/PyNET-PyTorch/blob/b05d817408cf729582b52a3fee214775ccab800c/model.py#L51) implementation while in the [Tensorflow](https://github.com/aiff22/PyNET/blob/16eeb71d94fb0c6ed40a5403ca5efd762974b2bf/model.py#L79) implementation the activations used are Tanh?
